ethpm_cli/commands/registry.py

```diff
@@ -203,3 +216,15 @@ def generate_registry_store_data(
         "alias": alias,
         "active": activate,
     }
+
+
+def display_packages(all_package_names: Iterable[str], config: Config) -> None:
+    for package_name in all_package_names:
+        all_releases = config.w3.pm.get_all_package_releases(package_name)
+        cli_logger.info(f"Retrieving all releases for {bold_blue(package_name)}: \n")
+        for version, manifest_uri in all_releases:
+            cli_logger.info(f"{bold_green(version)} --- ({bold_white(manifest_uri)})")
+        cli_logger.info(
+            f"Total releases: {config.w3.pm.get_release_count(package_name)}\n"
+        )
+    cli_logger.info(f"Packages in the registry: {config.w3.pm.get_package_count()}\n")
```

Comment:
Any particular reason that this comes at the end of the function? It might be more useful at the beginning - like if there are 1000 packages available, it'd be better to know before all of the calls for each particular package rather than after all 1000 calls have been made and displayed.

Reply:
No particular reason but you're suggestion makes a lot more sense. I'm not going to add a test for this line though to prevent it from being flaky in the future